liquidctl/driver/commander_pro.py

```diff
@@ -444,8 +446,8 @@ def set_color(self, channel, mode_str, colors, direction='forward', speed='mediu
 
         direction = _LED_DIRECTION_FORWARD if direction == 'forward' else _LED_DIRECTION_BACKWARD
         speed = _LED_SPEED_SLOW if speed == 'slow' else _LED_SPEED_FAST if speed == 'fast' else _LED_SPEED_MEDIUM
-        start_led = clamp(start_led, 1, 96) - 1
-        num_leds = clamp(maximum_leds, 1, 96-start_led-1)  # there is a current firmware limitation of 96 led's per channel
+        start_led = clamp(start_led, 1, 255) - 1
+        num_leds = clamp(maximum_leds, 1, 255 - start_led - 1)
```

Comment:
I think we can use 136 (4x34) for now.

Reply:
I think it might actually be 136, but I cant seem to find where that was documented again so I figured Id just limit it to one byte. I dont think anything bad will happen if this number is too big.

Reply:
I tend to prefer to be more strict in places like this one (easy fixes if necessary, but and the code more clearly states our [current] beliefs about what works and what doesn't), but I suppose it almost doesn't make a difference.  Feel free to keep it 255 (but I think it's fine to clamp it to 136 and, if necessary, change this again in the future).

Reply:
So that number was wrong too, based on [this](https://forum.corsair.com/v3/showthread.php?t=173880) link there can be up to 6 QL fans with 34 LED's each (a total of 204 LEDs). So I am updating the value to that.